### PR TITLE
hwdb: Fix touchpad enable/disable key

### PR DESCRIPTION
**Pull request by carlocaione**
https://phabricator.endlessm.com/T12104

### CODE REVIEW

File: hwdb/60-keyboard.hwdb [hunk 1]
```diff
@@ -492,9 +492,13 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHP440G2:pvr*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPProBook445G1NotebookPC:pvr*
 # HP ProBook 450 G0
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPProBook450G0:pvr*
+ KEYBOARD_KEY_81=f20                                    # Fn+F8; Microphone mute button, should be micmute
+
```
Comment (jprvita) [inline hwdb/60-keyboard.hwdb]: This should have been part of the previous commit, we both missed it on the 2nd round of review I guess. Can you please move this change to a separate commit, with a note to squash it on the original during the next rebase?

Comment (carlocaione) [inline hwdb/60-keyboard.hwdb]: nope, it's just that the diff algorithm is stupid in this case. I just moved all the `KEYBOARD_KEY_` on a separate section for the ProBook. The `+` should be on line 499, not on 495, but diff is messing this up.

Comment (jprvita) [inline hwdb/60-keyboard.hwdb]: Yeah, sorry, I got confused by the several models grouped together in a switch-case fashion. CR pass.
